src/pages/TeamPage/TeamPage.css

```
@@ -57,4 +59,99 @@
 
   .spacer {
     height: 16px; 
-  }
\ No newline at end of file
+  }
+
+  /* Contact Us Title */ 
+  .contact-title {
+    color: #00355E;
+    font-size: 30px;
+    font-weight: bold;
+    text-align: center;
+  }
+
+  .email-link {
+    color: black; 
+  }
+
+  /* Contact Us Email Form */ 
+  .card-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: 20px;
+  }
+
+  .form-group {
+    width: 45%;
+  }
+
+  .form-group label {
+    display: block;
+    margin-bottom: 5px;
+    justify-content: left;  
+    text-align: left;  
+  }
+
+  .form-group button {
+    background-color: blue;
+    color: white;
+    border: none;
+    padding: 10px 20px;
+    border-radius: 5px;
+    cursor: pointer;
+  }
+  
+  .form-group input {
+    border: none; 
+    width: 100%;
+    padding: 5px;
+    border-color: transparent; 
+    outline: none; 
+    border-bottom: 1px solid black; 
+  }
+
+  .form-group textarea {
+    resize: vertical;
+    width: 100%;
+    padding: 5px;
+  }  
+
+  .form-group-message {
+    width: 80%;
+    display: flex;
+    flex-direction: column;
+    margin: auto; 
+  }
+
+  .form-group-message label {
+    display: block;
+    margin-bottom: 5px;
+    text-align: left;
+  }
+
+  .form-group-message button {
+    background-color: #6599CC;
+    color: white;
+    border: none;
+    padding: 2% 2%; 
+    border-radius: 5px;
```

```suggestion
    border-radius: 30px;
```

A bit closer to the original figma design. I will walk through how to find these values in tonight's meeting!